ResetEra/dist/metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "author": {
+      "name": "fror",
+      "id": "515668127829458945"
+    },
+    "service": "ResetEra",
+    "description": {
+      "en": "ResetEra is the internet's premier video gaming forum destination. A savory assortment of enthusiasts, journalists, & developers."
+    },
+    "url": [
+      "www.resetera.com"
+    ],
```

Comment:
url shouldnt be in an array if its only 1

Reply:
okay